src/fluree/db/json_ld/api.cljc

```diff
@@ -169,6 +169,10 @@
   [db]
   (dbproto/-default-context db))
 
+(defn default-context-at-t
```

Comment:
I'm a bit torn on adding a new top-level function to do this, as I think the dx is a bit confusing. On the other hand, passing in a ledger instead of a db to `default-context` makes it harder to just pull out the default context from whatever db value I have at hand.

I'm not sure what the right call is, but I'm definitely biased towards having a smaller top-level api. What do you think?

Reply:
I'm not sure I follow the objection / question. Unlike `default-context` which operates on db values, this needs to operate on ledgers b/c `t` is a ledger-level concern. Previously we had several API fns that took ledgers or dbs but we moved away from that for reasons that I think still make sense. Does that address it?

Reply:
It's just that going by names `default-context-at-t` feels like it could be an opt to `default-context`. _I_ as a core developer understand the distinction between a ledger and a db, but a user may not. Maybe it would be simpler to have `default-context` take a ledger instead of a db, and have an opt for `t`, so that a user doesn't have to understand the distinction and can just get what they want.

Reply:
Yeah it's a little counterintuitive at first glance.

But I think that users will be better served by being encouraged to understand the ledger / db distinction from the get-go, or they're going to be confused on an ongoing basis. And the API should be clear about the distinction.

The names could potentially be improved, but I'm in favor of db-level fns taking db values and ledger-level fns taking ledgers, exclusively in both cases. There are plenty of situations where a db value is all you have and you'd find yourself grabbing its ledger only for the fn you're passing that to to immediately ask the ledger for its latest db. And supporting either/or makes for pretty messy, buggy code.

In an ideal world they could also just get a db at a certain `t` and then get its default-context using the existing fn. That doesn't work today and my attempt to make it work got a little hairy so I backtracked. But if we got that working we might not even need this as an API fn.

Reply:
This makes me feel like we might benefit from two top-level namespaces, one for dbs and one for ledgers, though I guess that could have its own complications.

If we are adding a top level api function, though, we should also add it to the `sdk` namespaces.

Reply:
OK, I can do that. The db-level `default-context` fn wasn't exposed there either so I'm adding that too.